workflow/scripts/functional_enrichment.py

```diff
@@ -146,3 +148,28 @@ def get_options():
     else:
         f = open(options.go, 'w')
         f.close()
+
```

Comment:
Could this be also repeated for Kegg pathways and Kegg modules? I guess the output could all be in one file

Reply:
ah ok, the KEGG_ko thing is the pathways? if so it should be fine like this